serializable.ts

```diff
@@ -32,7 +32,7 @@ export declare interface ToJSON {
 
 /** reutrns  a new javascript object with transformations */
 export declare interface FromJSON {
-  fromJSON(json: string | JSONValue | Object): this;
+  fromJSON(json: string | JSONValue | unknown): this;
```

Comment:
Personally I think it's safe to remove the `unknown` type for `json`. In my view this is either a string, or a json object.

In the implementation either it's a json string, or it's a valid json value. If someone wants to pass something else in they will have to cast it, or convert it themselves IMO
https://github.com/GameBridgeAI/ts_serialize/pull/130/files#diff-f9b7e368cb276079b7e49188cda17b6e719cd1501ba7f865ef47b84064c46e18L175